test/integration/nss-tests.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-2-Clause
+
+set -xo pipefail
+
+CA_PEM="$TEST_FIXTURES/ca.pem"
+CA_KEY="$TEST_FIXTURES/ca.key"
+
+CLIENT_CNF="$TEST_FIXTURES/client.cnf"
+PASSWORD_CA=whatever
+EXT_FILE="$TEST_FIXTURES/smimeextensions"
+
+export NSS_DEFAULT_DB_TYPE=sql
+
+if ! command -v certutil ||
+   ! command -v modutil; then
+  # Skip this test unless certutil/modutil are found
+  exit 77
```

Comment:
Other tests like [`pkcs11-tool.sh`](https://github.com/tpm2-software/tpm2-pkcs11/blob/master/test/integration/pkcs11-tool.sh) just give a hard failure if the required tools are not found. I think this is preferable since it guarantees that tests are not accidentally skipped due to a misconfigured build environment.

I suggest making this check return a hard error (`exit 99`) and adding an `AC_CHECK_PROG` for `certutil` and `modutil` to `configure.ac`: https://github.com/tpm2-software/tpm2-pkcs11/blob/2dbe1ffc582c9aa8521317e1e9eb5daa39ef4c80/configure.ac#L112-L132 Additionally these tools should be included in the [`INSTALL.md`](https://github.com/tpm2-software/tpm2-pkcs11/blob/master/docs/INSTALL.md#executable-dependencies) instructions.

Reply:
@diabonas he actually points this out in his message that its a TODO item, I plan on fixing it when I pull it in. I plan on working on this next week, im a bit behind.